### PR TITLE
fix: update headers

### PR DESCRIPTION
**Pull request by ds-ext-akononenko**
Update license header

I've used this template https://github.com/catenax-ng/foss-example/blob/main/catenax-ng-repositories/CopyrightAndLicenseHeader_1.txt

### CODE REVIEW

File: src/app/modules/shared/components/card-icon/card-icon.component.scss [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+card--icon {
```
Comment (evia-fbedarf) [inline src/app/modules/shared/components/card-icon/card-icon.component.scss:20]: . is missing

File: src/app/modules/shared/components/card-list/card-list.component.scss [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+card-list--container {
```
Comment (evia-fbedarf) [inline src/app/modules/shared/components/card-list/card-list.component.scss:20]: . is missing

File: src/app/modules/shared/components/sidenav/sidenav.component.scss [hunk 3]
```diff
@@ -1,22 +1,23 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+sidenav {
```
Comment (evia-fbedarf) [inline src/app/modules/shared/components/sidenav/sidenav.component.scss:20]: . is missing

File: src/app/modules/shared/modules/relations/presentation/part-relation.component.scss [hunk 4]
```diff
@@ -1,22 +1,23 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+part-relation--button-container {
```
Comment (evia-fbedarf) [inline src/app/modules/shared/modules/relations/presentation/part-relation.component.scss:20]: . is missing

**Review by evia-fbedarf — CHANGES_REQUESTED** [review]
Please Check all files again, I don't know if I got all the missing dots.

**Comment by ds-ext-akononenko** [comment]
I've fixed scss files. There was wrong format

**Review by evia-fbedarf — APPROVED** [review]
LGMT

**Pull request merged**